Vic2ToHoI4/Data_Files/configurables/Vic2Localisations/dominions.csv

```diff
@@ -1,14 +1,13 @@
-;en;fr;de;pl;es;it;sw;cz;hu;nl;pt;ru;fi;
-# Localisations for generated dominions;;;;;;;;;;;;;;
-# The converter tries to find localisations in this order:;;;;;;;;;;;;;;
-#   * dom_<culture group>_<ideology>_<region>;;;;;;;;;;;;;;
-#   * dom_<culture group>_<region>;;;;;;;;;;;;;;
-#   * dom_<culture>_<ideology>_<region>;;;;;;;;;;;;;;
-#   * dom_<culture>_<region>;;;;;;;;;;;;;;
-#   * dom_<ideology>_<region>;;;;;;;;;;;;;;
-#   * dom_<region>;;;;;;;;;;;;;;
-# If no localisation is found, the final localisation will be "$OWNER_ADJ$ $REGION$";;;;;;;;;;;;;;
-#;;;;;;;;;;;;;;

```

Comment:
I think this line was/is used as a template - copy it 5 times then fill in the columns, to make sure you didn't miss a column.

Reply:
There's a better one below that I've made more prominent.